web/models/workflow.ex

```diff
@@ -29,9 +29,9 @@ defmodule Designator.Workflow do
     query = from sms in "set_member_subjects",
       left_join: swc in "subject_workflow_counts", on: (sms.subject_id == swc.subject_id and swc.workflow_id == ^workflow_id),
       where: sms.subject_set_id == ^subject_set_id and is_nil(swc.retired_at),
-      select: {sms.subject_id, sms.priority}
+      select: { sms.subject_id, fragment("CASE WHEN priority IS NULL THEN 0 ELSE priority END") }
```

Comment:
I'm assuming the `fragment` part is to take care of the case when priority is `nil`?  Does the sort/Decimal compare have issues when comparing against `nil`?